src/components/Patient/PatientDetailsTab/EncounterHistory.tsx

```diff
@@ -4,53 +4,24 @@ import { useTranslation } from "react-i18next";
 import PaginatedList from "@/CAREUI/misc/PaginatedList";
 
 import CircularProgress from "@/components/Common/CircularProgress";
-import Loading from "@/components/Common/Loading";
 import { ConsultationCard } from "@/components/Facility/ConsultationCard";
 import { ConsultationModel } from "@/components/Facility/models";
+import { PatientProps } from "@/components/Patient/PatientDetailsTab";
+import { PatientModel } from "@/components/Patient/models";
 
-import useAuthUser from "@/hooks/useAuthUser";
-
-import { triggerGoal } from "@/Integrations/Plausible";
 import routes from "@/Utils/request/api";
-import useTanStackQueryInstead from "@/Utils/request/useQuery";
-
-import { PatientProps } from ".";
-import { PatientModel } from "../models";
 
 const EncounterHistory = (props: PatientProps) => {
-  const { patientData: initialPatientData, facilityId, id } = props;
+  const { patientData: initialPatientData, id, refetch } = props;
   const [patientData, setPatientData] =
     useState<PatientModel>(initialPatientData);
```

Comment:
Why do we need a state that just takes in the props on mount and never change it in the lifecycle of the component?

```suggestion
  const { patientData, id, refetch } = props;
```

Reply:
Done the changes